readthedocs-theme/templates/index.html

```diff
@@ -0,0 +1,113 @@
+{% extends "base.html" %}
+{# This is the blog index, not the home page template #}
+
+
+{% block content %}
+
+  <!-- Intro -->
+  <section class="ui vertical segment" id="blog-intro">
+    <div class="ui container">
+      <div class="ui segment basic">
+
+        {% block content_title %}
+          <div class="ui basic vertical big segment">
+          <h1 class="ui header">
+            <span class="ui header">Read the Docs Blog</span>
+          </h1>
+            <p>
+              Short intro
+            </p>
+          </div>
+        {% endblock %}
+
+      </div>
+    </div>
+  </section>
+
+
+  <section class="ui vertical segment" id="articles-list">
+    <div class="ui container">
+      <div class="ui segment basic">
+
+        <!-- Latest  2 articles -->
+        <div class="ui two cards stackable">
+          {% for article in articles_page.object_list %}
+            {% if not articles_previous_page and loop.index <= 2 %}
+              <a class="ui card" href="/{{ article.url }}">
```

Comment:
The text is all underlined for me, which is not the normal styles for cards:

![image](https://user-images.githubusercontent.com/1140183/152938725-f664983f-42d3-4ed2-be93-c9ff8ca4aab3.png)

I believe this needs an extra nested `div`, see: https://semantic-ui.com/views/card.html#link-card

Reply:
* Opened a PR to fix this in #69